django/test/utils.py

```diff
@@ -55,6 +56,16 @@
 TZ_SUPPORT = hasattr(time, "tzset")
 
 
+# RemovedInDjango50Warning
+class DjangoTableFormRenderer(DjangoTemplates):
+    form_template_name = "django/forms/table.html"
```

Comment:
Once the new div renderer is added, this (and the Jinja version) shouldn't be needed? 🤔

(Interesting how many tests depend on the table output...)